MentoringHomeWork/HW6/HW6.Task2/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using HW6.Task2;
+
+NewArrayPosition ar = new NewArrayPosition();
+NewArrayPosition.AddNewPosition();
```

Comment:
![image](https://github.com/user-attachments/assets/fa03ab5f-bf60-4c64-b69b-5a5b96a186f2)
I didn't enter the number '0' - could you try to resolve it?

Reply:
resolved